src/iso19111/operation/transformation.cpp

```diff
@@ -366,7 +366,7 @@ TransformationNNPtr Transformation::create(
 
 // ---------------------------------------------------------------------------
 
-/** \brief Instantiate a transformation ands its OperationMethod.
+/** \brief Instantiate a transformation and it's OperationMethod.
```

Comment:
that one should be reverted

Reply:
oops no, I see that the actual fix was ands->and.  I meant that the it's should be its